### PR TITLE
Remove unnecessary sprintf in node status tests

### PR DESCRIPTION
**Pull request by saschagrunert**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

There is no invocation to sprintf needed for those strings so we can
remove them.

**Which issue(s) this PR fixes**:

None

**Special notes for your reviewer**:

None

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm
/retest

File: pkg/kubelet/kubelet_node_status_test.go [hunk 1]
```diff
@@ -224,31 +224,31 @@ func TestUpdateNewNodeStatus(t *testing.T) {
...
-							Message:            fmt.Sprintf("kubelet has sufficient PID available"),
+							Message:            "kubelet has sufficient PID available",
 							LastHeartbeatTime:  metav1.Time{},
 							LastTransitionTime: metav1.Time{},
 						},
 						{
 							Type:               v1.NodeReady,
 							Status:             v1.ConditionTrue,
 							Reason:             "KubeletReady",
-							Message:            fmt.Sprintf("kubelet is posting ready status"),
```
Comment (tanjunchen) [inline pkg/kubelet/kubelet_node_status_test.go:251]: nice catch!

**Comment by derekwaynecarr** [comment]
perils of copy/paste.

/approve
/lgtm

**Pull request merged**